docs/website/.eleventy.cjs

```diff
@@ -0,0 +1,100 @@
+const litPlugin = require('@lit-labs/eleventy-plugin-lit');
+const eleventyNavigationPlugin = require('@11ty/eleventy-navigation');
+const markdownItAnchor = require('markdown-it-anchor');
+const markdownIt = require('markdown-it');
+const syntaxHighlight = require('@11ty/eleventy-plugin-syntaxhighlight');
+const slugify = require('slugify');
+const htmlMinifier = require('html-minifier');
+
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPlugin(litPlugin, {
+    mode: 'worker',
+    componentModules: [
+      './src/js/demo-greeter.js',
+      './src/js/my-element.js',
+    ],
+  });
+
+  eleventyConfig.addPlugin(syntaxHighlight);
+
+  eleventyConfig.addPlugin(eleventyNavigationPlugin);
+
+  function removeExtraText(s) {
+    let newStr = String(s).replace(/New\ in\ v\d+\.\d+\.\d+/, '');
+    newStr = newStr.replace(/Coming\ soon\ in\ v\d+\.\d+\.\d+/, '');
+    newStr = newStr.replace(/⚠️/g, '');
+    newStr = newStr.replace(/[?!]/g, '');
+    newStr = newStr.replace(/<[^>]*>/g, '');
+    return newStr;
+  }
+
+  function markdownItSlugify(s) {
+    return slugify(removeExtraText(s), { lower: true, remove: /[:’'`,]/g });
+  }
+
+  const anchor = require('markdown-it-anchor');
+
+  let mdIt = markdownIt({
+    html: true,
+    breaks: true,
+    linkify: true,
+  })
+    .disable('code')
+    .use(markdownItAnchor, {
+      permalink: anchor.permalink.headerLink(),
+      slugify: markdownItSlugify,
+      permalinkBefore: true,
+      permalinkClass: 'direct-link',
+      permalinkSymbol: '#',
+      level: [1, 2, 3, 4],
+    });
+
+  eleventyConfig.setBrowserSyncConfig({
+    notify: true,
+    files: './public/css/**/*.css',
+  });
+
+  eleventyConfig.setLibrary('md', mdIt/*markdownIt(options)*/);
+
+  eleventyConfig.addPassthroughCopy('./src/assets');
+
+  const fs = require('fs');
```

Comment:
I prefer these at the top. That way you don't have to search where it is defined.